src/Xrefcheck/Progress.hs

```diff
@@ -137,18 +137,18 @@ showProgress name width col posixTime Progress{..} = mconcat
   , status
   ]
   where
-    -- | Each of the following values represents the number of the progress bar cells
+    -- Each of the following values represents the number of the progress bar cells
```

Comment:
I was getting some warning regarding to Haddock comments not allowed in where bindings.